invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -33,6 +33,7 @@
     EDTFDateString,
     EDTFDateTimeString,
     IdentifierSet,
+    IdentifierValueSet,
```

Comment:
You can remove the unused `IdentifierSet` above

Reply:
it is used in authors identifiers. There is makes sense to keep the deduplication by schema (as we said yesterday)